dev/tidb-in-kubernetes/troubleshoot.md

```diff
@@ -332,3 +332,25 @@ Normally, when a TiKV Pod is in a healthy state (`Running`), the corresponding T
         ```
 
     After the Pod is re-created, a new store is registered in the TiKV cluster. Then the recovery is completed.
+
+## Long queries are abnormally interrupted in TiDB
+
+Load Balancers set the idle connection timeout. If no data is sent over a connection for a specific period of time, Load Balancer closes the connection.
+
+If a long query is interrupted when you use TiDB, check the middleware application between the client and the TiDB server.
+If you find that the idle timeout is not long enough for your query, try to set the timeout to a larger value. If you cannot reset it, enable the `tcp-keep-alive` option.
+
+In Linux, the keepalive packets are sent at the interval of 7,200 seconds by default. To shorten the interval, configure `sysctls` via the `podSecurityContext` field. Here is an example:
```

Comment:
The `tcp_keepalive_time` interval is not between keepalive packets. It is between the first probe packet and the last data packet. 

The first keepalive probe packet is sent after 7,200 seconds since the last data packet by default.